google-analytics-data-v1alpha/lib/google/analytics/data/v1alpha/alpha_analytics_data/client.rb

```diff
@@ -0,0 +1,760 @@
+# frozen_string_literal: true
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Auto-generated by gapic-generator-ruby. DO NOT EDIT!
+
+require "google/cloud/errors"
+require "google/analytics/data/v1alpha/analytics_data_api_pb"
+
+module Google
+  module Analytics
+    module Data
+      module V1alpha
+        module AlphaAnalyticsData
```

Comment:
It really seems unusual and incorrect to include "Alpha" in this module name. Is there anything that can be done about this?

Reply:
Ugh, thanks for catching that. Yes, we do have a generator switch specifically for changing bad service names like that. I'll apply it and regen...